blog/_posts/GSoC 2017 project A Wrapper for the FEniCS Finite Element Toolbox.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: post
+title: "GSoC 2017 : A Wrapper for the FEniCS Finite Element Toolbox"
+authors: Yiannis Simillides,Bart Janssens, Chris Rackauckas
+---
+
+<script type="text/javascript"
+  src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML">
+</script>
+
+
+# Introduction
+
+Throughout this Google Summer of Code project I, along with my mentors, aimed to create a Wrapper for the FEniCS Finite Element Toolbox in the Julia Language. Our work done can be found at [FEniCS.jl](https://github.com/JuliaDiffEq/FEniCS.jl) . This would allow users to perform FEM calculations directly in Julia, utilizing our [PyCall.jl](https://github.com/JuliaPy/PyCall.jl) wrapping functionality. We currently have wrapped the main  functionality, along with providing the necessary instructions to add further components when they are deemed necessary. Members of the Julia community not directly related to the project also contributed small fixes and suggestions throughout the project. The majority of the code produced has already been merged to the GitHub repository (which was created specifically for this project). One of the main improvements which would greatly increase its usage would be the further integration with the JuliaDiffEq package. 
+
+# What is FEniCS?
+
+**[FEniCS](https://fenicsproject.org/)** describes itself as a popular open-source (LGPLv3) computing platform for solving partial differential equations (PDEs). FEniCS enables users to quickly translate scientific models into efficient finite element code. In layman's terms it lets us solve math problems.
+
```

Comment:
The last sentence here is a bit too general, maybe something like "it lets us write complicated mathematical equations and solves them automatically using computer simulation"